platform/ios/ios.cmake

```diff
@@ -73,7 +73,9 @@ target_include_directories(
 )
 
 include(${PROJECT_SOURCE_DIR}/vendor/icu.cmake)
-# include(${PROJECT_SOURCE_DIR}/platform/ios/ccache.cmake)
+if(MBGL_IOS_CCACHE)
```

Comment:
Just a nit, this would be better if we had an option in the `ios.cmake`

`option(MBGL_WITH_IOS_CCACHE "Enable ccache for iOS" OFF)`

Also use the `_WITH_` pattern. Otherwise LGTM.

Reply:
Done